src/main/java/io/debezium/connector/vitess/VitessValueConverter.java

```diff
@@ -42,6 +42,9 @@ public SchemaBuilder schemaBuilder(Column column) {
         if (matches(typeName, Query.Type.ENUM.name())) {
             return io.debezium.data.Enum.builder(column.enumValues());
         }
+        if (matches(typeName, Query.Type.SET.name())) {
+            return io.debezium.data.EnumSet.builder(column.enumValues());
```

Comment:
I overloaded the `builder()`  function in the core PR: https://github.com/debezium/debezium/pull/2167. The CI build will pass once the core PR is merged.